raster/src/main/scala/geotrellis/raster/render/ColorMap.scala

```diff
@@ -259,6 +263,13 @@ class IntColorMap(breaksToColors: Map[Int, Int], val options: Options = Options.
     h.foreachValue(z => ch.setItem(z, map(z)))
     new IntCachedColorMap(orderedColors, ch, options)
   }
+
+  lazy val breaksString: String = {
```

Comment:
For the same reason that `orderedColors` is lazy, so that these potentially unneeded values aren't calculated upon `ColorMap` instantiation.